docs/helping.md

```diff
@@ -1 +1,85 @@
+
+Generally, you _should_ use `/closerequest` for closing a ticket. There's much less chance the end user gets upset, and it can be denied last minute if another issue comes up.\
+![What using the `/closerequest` command looks like](images/closerequest.png)
+
+From here, the user can accept the close request, which in turn closes the ticket, or deny the close request, while results in an edited embed telling you they denied it.
+
+To _use_ the `/closerequest` command, there are a few "ways" to use it.\
+Note that `close_delay` is a number, meaning the time in hours before the ticket autocloses.\
+`reason` is a box to input the reason for a close request.\
+Using "ac" as described later lets other people know how long until the ticket will close ("ac" meaning **A**uto **C**lose). This helps yourself and others when looking at tickets because the tickets bot doesn't show when a ticket is set to auto close.
+
+If the ticket seems resolved/the user has said they have no issues, use `/closerequest` with `close_delay` set to `8` and the `reason` set to "resolved, ac 8hr"
+
+For tickets where the end user that opened the ticket _never_ responded, we usually wait a couple days, ping them a few times, and if they continue to be inactve, set a close request with `close_delay` set to 24 and the `reason` set to "no response, ac 24hr"
+For inactive tickets (when it's the _user_ not responding, not us!) we usually give them a few days and then start pinging them once a day. 
+If they continue to not respond for 2-3 days, we tend to set a close request with `close_delay` set to `48` and the `reason` set to  "inactive ticket, ac 48hr"
+
+You can also use `/closerequest` without giving a `close_delay` or a `reason`, however you should always try to give a reason for closing a ticket, and if you don't give a `close_delay` there's a high chance that the ticket doesn't get closed (quite a few end users don't accept or see the close request)
+
+You can view information on the closed tickets, such as open and close date, who opened the ticket, who closed the ticket, and the reason for closing the ticket in the `#transcripts` channel. The bot also provides an online log of the entire ticket connected to the embed when a ticket is closed.
+
+## Miscellaneous 
+
+The Northstar Discord server also has a couple custom bots in it. These are:
+
+birb and H0L0's bot [Northstar Servers](https://github.com/hummusbird/northstar-bot), which allows you to run commands like `.status` and see how many people are online on Northstar currently, and things such as `.search region aus`, which would search servers with the region Australia. You can check the repo for more info.
+
+Cyn's bot [Spectre](https://github.com/itscynxx/Spectre) automatically replies to specific error messages, which you can see a list of [here](https://github.com/itscynxx/Spectre/wiki/Replies)
```

Comment:
Could make subsections for those or a list so that it's formatted in a more readable way \^\^

Reply:
Done with 54c9ebda40176bfe23836bb45890b881b4f5f61a